### PR TITLE
Darter Pro 10 (darp10)

### PR DESCRIPTION
**Pull request by jacobgkau**
This adds the Darter Pro 10 to tech docs. The 16" model (darp10) and the 14" model (darp10-b) are both covered in the new section.

### CODE REVIEW

File: src/models/darp10/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        - Intel Wi-Fi 6E AX210/AX211
+            - Bluetooth 5.3
+- Power
+    - 90W (19V, 4.74A) DC-in port
+        - Barrel size: 5.5mm (outer), 2.5mm (inner)
+    - Included AC adapter: Chicony A16-090P1A
+        - AC power cord type: IEC C5
+    - USB-C charging compatible with 90W+ charger
+    - 73Wh 4-cell Lithium-ion battery
+        - Model number: L140BAT-4
```
Comment (leviport) [inline src/models/darp10/README.md:50]: I believe the battery model number is `V540-BAT-4-73` now. Should be the same on both sizes.

File: src/models/darp10/repairs.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+### Steps to replace the keyboard:
+
+1. Place the machine lid-side down.
+    - Use a soft surface (such as a towel) to avoid scratches.
+2. Remove the keyboard screws (highlighted yellow below).
+    - On the 16" model (darp10), there is one keyboard screw.
+    ![Keyboard screws (16")](./img/keyboard-screw-16.webp)
+    - On the 14" model (darp10-b), there are two keyboard screws.
+    ![Keyboard screws (14")](./img/keyboard-screws-14.webp)
+3. For the 16" model (darp10) only, you can optionally [remove the bottom panel](), [remove the heatsink](), and slowly pull the tabs highlighted below to remove the two keyboard adhesive strips.
```
Comment (leviport) [inline src/models/darp10/repairs.md:239]: These bottom panel and heatsink links aren't populated

File: src/models/darp10/repairs.md [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+    ![Keyboard ribbons (16")](./img/keyboard-ribbons-16.webp)
+    - On the 14" model (darp10-b), the keyboard backlight cable is to the left of the main cable.
+    ![Keyboard ribbons (14")](./img/keyboard-ribbons-14.webp)
+7. Flip the black latches upwards to free the ribbon cables.
+8. Pull the ribbon cables out of the connectors.
+9. Remove the keyboard and replace it with the new one.
+10. Carefully slide both ribbon cables into their connectors.
+11. For the 16" model (darp10) only, optionally replace the keyboard adhesive strips (if they were removed earlier).
+    - The keyboard can be installed without adhesive strips if they're no longer usable.
+12. Flip the black latches back into place to secure the ribbon cables.
```
Comment (leviport) [inline src/models/darp10/repairs.md:258]: Should steps 11 and 12 be swapped? I feel like cable latches should follow immediately after plugging the cables into the sockets.

**Review by leviport — CHANGES_REQUESTED** [review]
A couple small things, but this looks great!

File: src/models/darp10/repairs.md (hunk 2 above)
Comment (jacobgkau) [reply src/models/darp10/repairs.md:239]: Fixed.

File: src/models/darp10/repairs.md (hunk 3 above)
Comment (jacobgkau) [reply src/models/darp10/repairs.md:258]: Yes.

File: src/models/darp10/README.md (hunk 1 above)
Comment (jacobgkau) [reply src/models/darp10/README.md:50]: Updated.

**Review by leviport — APPROVED** [review]
:metal:

**Comment by jacobgkau** [comment]
As always, support's welcome to comment for future changes when they have time. I want to get this out since we've had people asking for it, though.

**Pull request merged**